Apache Traffic Server - document changes. (#14203)
newrelic/docs-website: src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx

If you need to install the integration in a different way, see [Apache monitoring integration](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/apache-monitoring-integration/).

## Expose your metrics
After successful installation, Apache Traffic Server has to be started successfully, then it should start running on a `your_ip` with port `8080`. You need to configure the metrics exposed stats.

1. Go to the `plugin.config` file:
  ```shell
  sudo nano /etc/trafficserver/plugin.config
  ```
2. Add the below line in your `plugin.config` file.
  ```
  stats_over_http.so
  ``` 
3. Make sure your metrics are exposed on the below URL.

   Metrics of your Apache Traffic Server: `http://YOUR_IP:8080/_stats`

## Configure NRI-Flex for Apache Traffic Server [#integrate]

Flex comes bundled with the New Relic infrastructure agent. To create a flex configuration file follow these steps:

> 1. Creates files named `nri-flex-apache-traffic-server-config.yml`, `nri-flex-ats-cache-config.yml`, and `nri-flex-ats-http-config.yml` in this path:

Reviewer: ```suggestion
1. Create three files named `nri-flex-apache-traffic-server-config.yml`, `nri-flex-ats-cache-config.yml`, and `nri-flex-ats-http-config.yml` in this path:
```

## Install Apache New Relic Agent
Our Apache integrations collects network metrics like TCP connections, DNS lookup, HTTPS, SSL, and server and worker statuses.
To install the integration, follow the steps in our [guided install](https://one.newrelic.com/launcher/nr1-core.explorer?pane=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5saXN0aW5nIn0=&cards%5B0%5D=eyJuZXJkbGV0SWQiOiJucjEtaW5zdGFsbC1uZXdyZWxpYy5ucjEtaW5zdGFsbC1uZXdyZWxpYyIsInBhdGgiOiJvaGkiLCJyZWNpcGVOYW1lIjoiYXBhY2hlLW9wZW4tc291cmNlLWludGVncmF0aW9uIiwiYWN0aXZlQ29tcG9uZW50IjoiVlRTT0NvbW1hbmQiLCJhY3RpdmVFbnZpcm9ubWVudCI6ImFwYWNoZS1vcGVuLXNvdXJjZS1pbnRlZ3JhdGlvbiJ9).
If you need to install the integration in a different way, see [Apache monitoring integration](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/apache-monitoring-integration/).

## Expose your metrics
After successful installation, Apache Traffic Server has to be started successfully, then it should start running on a `your_ip` with port `8080`. You need to configure the metrics exposed stats.

> Goto the `plugin.config` file.

Reviewer: ```suggestion
1. Go to the `plugin.config` file.
```

Author: Hi @ally-sassman,
As per your suggestion, I updated the changes. Kindly review it.

Thanks.

  ```shell
  sudo nano /etc/trafficserver/plugin.config
  ```

> Add the following line `stats_over_http.so` in your `plugin.config` file.

Reviewer: @pkudikyala This is missing a code block - please add it here.

Author: Hi @ally-sassman,
As per your suggestion, I updated the code blocks. Kindly review it.

Thanks.

> After successful installation, Apache Traffic Server has to be started successfully, then it should start running on a `your_ip` with port `8080`. You need to configure the metrics exposed stats.
> 
> Goto the `plugin.config` file.
>   ```shell
>   sudo nano /etc/trafficserver/plugin.config
>   ```
> Add the following line `stats_over_http.so` in your `plugin.config` file.

Reviewer: ```suggestion
After successful installation, Apache Traffic Server has to be started successfully, then it should start running on a `your_ip` with port `8080`. You need to configure the metrics exposed stats.

Goto the `plugin.config` file.
  ```shell
  sudo nano /etc/trafficserver/plugin.config
  ```
2. Add the following line `stats_over_http.so` in your `plugin.config` file.
```

> Add the following line `stats_over_http.so` in your `plugin.config` file.

Reviewer: ```suggestion
2. Add the following line `stats_over_http.so` in your `plugin.config` file.
```

Author: Hi @ally-sassman,
As per your suggestion, I updated the changes. Kindly review it.

Thanks.

> Make sure your metrics are exposed on the below URLs.

Reviewer: @pkudikyala What are the below URLs? Are they missing here?

Author: Hi @ally-sassman,
Here are no more URLs, I updated the description. Kindly review it.

Thanks.

> Make sure your metrics are exposed on the below URLs.

Reviewer: ```suggestion
2. Make sure your metrics are exposed on the below URLs.
```

Author: Hi @ally-sassman,
As per your suggestion, I updated the changes. Kindly review it.

Thanks.

> ## Install Apache New Relic Agent
> Our Apache integrations collects network metrics like TCP connections, DNS lookup, HTTPS, SSL, and server and worker statuses.
> To install the integration, follow the steps in our [guided install](https://one.newrelic.com/launcher/nr1-core.explorer?pane=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5saXN0aW5nIn0=&cards%5B0%5D=eyJuZXJkbGV0SWQiOiJucjEtaW5zdGFsbC1uZXdyZWxpYy5ucjEtaW5zdGFsbC1uZXdyZWxpYyIsInBhdGgiOiJvaGkiLCJyZWNpcGVOYW1lIjoiYXBhY2hlLW9wZW4tc291cmNlLWludGVncmF0aW9uIiwiYWN0aXZlQ29tcG9uZW50IjoiVlRTT0NvbW1hbmQiLCJhY3RpdmVFbnZpcm9ubWVudCI6ImFwYWNoZS1vcGVuLXNvdXJjZS1pbnRlZ3JhdGlvbiJ9).
> If you need to install the integration in a different way, see [Apache monitoring integration](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/apache-monitoring-integration/).
> 
> ## Expose your metrics
> After successful installation, Apache Traffic Server has to be started successfully, then it should start running on a `your_ip` with port `8080`. You need to configure the metrics exposed stats.
> 
> Goto the `plugin.config` file.
>   ```shell
>   sudo nano /etc/trafficserver/plugin.config
>   ```
> Add the following line `stats_over_http.so` in your `plugin.config` file.
> 
> Make sure your metrics are exposed on the below URLs.
> Metrics of name node: `http://YOUR_IP:8080/_stats`
> 

Reviewer: ```suggestion
## Install the New Relic Apache agent

Our Apache integrations collect network metrics like TCP connections, DNS lookup, HTTPS, SSL, and server and worker statuses.
To install the integration, follow the steps in our [guided install](https://one.newrelic.com/launcher/nr1-core.explorer?pane=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5saXN0aW5nIn0=&cards%5B0%5D=eyJuZXJkbGV0SWQiOiJucjEtaW5zdGFsbC1uZXdyZWxpYy5ucjEtaW5zdGFsbC1uZXdyZWxpYyIsInBhdGgiOiJvaGkiLCJyZWNpcGVOYW1lIjoiYXBhY2hlLW9wZW4tc291cmNlLWludGVncmF0aW9uIiwiYWN0aXZlQ29tcG9uZW50IjoiVlRTT0NvbW1hbmQiLCJhY3RpdmVFbnZpcm9ubWVudCI6ImFwYWNoZS1vcGVuLXNvdXJjZS1pbnRlZ3JhdGlvbiJ9).
If you need to install the integration in a different way, see [Apache monitoring integration](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/apache-monitoring-integration/).

## Expose your metrics

After successful installation, Apache Traffic Server has to be started successfully, then it should start running on a `your_ip` with port `8080`. You need to configure the metrics exposed stats.

1. Go to the `plugin.config` file.
  ```shell
  sudo nano /etc/trafficserver/plugin.config
  ```
2. Add the following line `stats_over_http.so` in your `plugin.config` file:

Make sure your metrics are exposed on the below URLs.
Metrics of name node: `http://YOUR_IP:8080/_stats`

```

## Configure NRI-Flex for Apache Traffic Server [#integrate]

Flex comes bundled with the New Relic infrastructure agent. To create a flex configuration file follow these steps:

> 1. Creates files with  named `nri-flex-apache-traffic-server-config.yml`, `nri-flex-ats-cache-config.yml` and `nri-flex-ats-http-config.yml` in this path:

Reviewer: ```suggestion
1. Creates files named `nri-flex-apache-traffic-server-config.yml`, `nri-flex-ats-cache-config.yml`, and `nri-flex-ats-http-config.yml` in this path:
```

Author: Hi @ally-sassman,
As per your suggestion, I updated the changes. Kindly review it.

Thanks.

  ```shell
  cd /etc/newrelic-infra/integrations.d
  ```

> 2. Your `nri-flex-apache-traffic-server-config.yml`, `nri-flex-ats-cache-config.yml` and `nri-flex-ats-http-config.yml` file should look like this:

Reviewer: ```suggestion
2. Your `nri-flex-apache-traffic-server-config.yml`, `nri-flex-ats-cache-config.yml`, and `nri-flex-ats-http-config.yml` file should look like this:
```

Author: Hi @ally-sassman,
As per your suggestion, I updated the code blocks. Kindly review it.

Thanks.

  The value on the `event_type` is used to store metrics on the NRDB.

>   File: `nri-flex-apache-traffic-server-config.yml`

Reviewer: ```suggestion
Your `nri-flex-apache-traffic-server-config.yml` file should look like this:
```

Author: Hi @ally-sassman,
As per your suggestion, I updated the changes. Kindly review it.

Thanks.

If you need to install the integration in a different way, see [Apache monitoring integration](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/apache-monitoring-integration/).

## Expose your metrics
After successful installation, Apache Traffic Server has to be started successfully, then it should start running on a `your_ip` with port `8080`. You need to configure the metrics exposed stats.

1. Go to the `plugin.config` file:
  ```shell
  sudo nano /etc/trafficserver/plugin.config
  ```
2. Add the below line in your `plugin.config` file.
  ```
  stats_over_http.so
  ``` 
3. Make sure your metrics are exposed on the below URL.

   Metrics of your Apache Traffic Server: `http://YOUR_IP:8080/_stats`

## Configure NRI-Flex for Apache Traffic Server [#integrate]

Flex comes bundled with the New Relic infrastructure agent. To create a flex configuration file follow these steps:

> 1. Go to the `integrations.d` file:

Reviewer: ```suggestion
1. Go to the `integrations.d` directory:
```
